nemo/collections/asr/parts/perturb.py

```diff
@@ -10,6 +10,13 @@
 from nemo.collections.asr.parts import collections, parsers
 from nemo.collections.asr.parts.segment import AudioSegment
 
+try:
+    from nemo.collections.asr.parts import numba_utils
+
+    HAVE_NUMBA = True
+except (ImportError, ModuleNotFoundError):
+    HAVE_NUMBA = False
+
```

Comment:
If numba is easy to install, just add it to requirements and remove check

Reply:
Numba is particularly hard to install using pip. On Windows, it doesn't work cause of llvmlite package. Conda is the preferred method of installing it, which is why I am doing an optional check.